File: python/2tuple/3rd.py
'''
 Write a Python program to create a tuple of n numbers and print maximum, minimum, and
sum of elements in a tuple.
'''

t=()
l=[]
n =int(input("enter the number:"))
for i in range(0,n):
    print(f"index {i+1}:")
    ele = int(input())
    l.append(ele)

t=tuple(l)

# orrrrrr
# t = (1,2,3)

maximum = max(t)
mini = min(t)
summ = sum(t)
print(f"tuple is :{t}")
print(f"maximum element :{maximum}")
print(f'minimum element :{mini}')
print(f"sum of elements :{summ}")

File: c++/c++ clg assignments/1/3rd.cpp
/*
.Write a C++ program to check if number is prime or not.
*/
#include<iostream>

using namespace std;

int main()
{
    int n;
    cout << "enter the number :";
    cin >> n;
    int isprime = 1;
    if(n ==0 && n==1)
    {
        isprime = 0;
    }
    for (int i = 2; i <= n/2; i++)
    {
        if(n% i==0){
            isprime = 0;
            break;
        }

    }
    if(isprime == 1)
        cout <<endl << n << " is a prime number";
    else
        cout <<endl<< n << " is not a prime number";

    return 0;
}